include/boost/fusion/container/generation/make_list.hpp

```diff
@@ -10,7 +10,37 @@
 #include <boost/fusion/support/config.hpp>
 #include <boost/fusion/container/list/list.hpp>
 
+#if !defined(BOOST_FUSION_HAS_VARIADIC_LIST)
 # include <boost/fusion/container/generation/detail/pp_make_list.hpp>
+#else
 
+///////////////////////////////////////////////////////////////////////////////
+// C++11 variadic interface
+///////////////////////////////////////////////////////////////////////////////
+
+#include <boost/fusion/support/detail/as_fusion_element.hpp>
+
+namespace boost { namespace fusion
+{
+    namespace result_of
+    {
+        template <typename ...T>
+        struct make_list
+        {
+            typedef list<T...> type;
+        };
+    }
+
+    template <typename ...T>
+    BOOST_CONSTEXPR BOOST_FUSION_GPU_ENABLED
+    inline list<typename detail::as_fusion_element<T>::type...>
+    make_list(T const&... arg)
```

Comment:
No, I meant [list](https://github.com/boostorg/fusion/blob/ddcd3cdf93d7d3abb96c093897f52ea07ab21ee3/include/boost/fusion/container/list/detail/cpp03/list.hpp) doesn't check `BOOST_NO_CXX11_RVALUE_REFERENCES` like [this](https://github.com/boostorg/fusion/blob/ddcd3cdf93d7d3abb96c093897f52ea07ab21ee3/include/boost/fusion/container/vector/detail/cpp03/vector.hpp#L159-L187).